src/main/java/io/jenkins/plugins/RestApp.java

```diff
@@ -20,13 +21,18 @@
 public class RestApp extends ResourceConfig {
 
   public RestApp() {
+    System.setProperty("sentry.stacktrace.app.packages", "io.jenkins.plugins");
```

Comment:
do you need this here and in the dockerfile?

Reply:
oh docker file doesn't work, i thought i deleted that line in the file